features/src/main/feature/camel-features.xml

```diff
@@ -2027,6 +2027,11 @@
         <bundle dependency='true'>wrap:mvn:org.springframework.ldap/spring-ldap-core/${spring-ldap-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-spring-ldap/${project.version}</bundle>
     </feature>
+    <feature name='camel-spring-main' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='${camel.osgi.spring.version}'>spring</feature>
```

Comment:
```suggestion
        <feature version='${camel.osgi.version.range}'>camel-spring</feature>
```
Let's depend on the `camel-spring` feature instead as suggested by the dependency:tree